spec/controllers/post_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe PostController, type: :controller do
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "GET #new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
```

Comment:
Shouldn't this be a `POST` route?

Reply:
These generated tests might be out sync with the actual code after making some manual changes. But I think this one is okay as a GET. You can visit localhost:3000/posts/new to get to the page with the new post form. I believe sending a POST to /posts will create a post